scripts/zones/Bhaflau_Thickets/IDs.lua

```diff
@@ -39,6 +39,25 @@ zones[xi.zone.BHAFLAU_THICKETS] =
     },
     mob =
     {
+        CHIGOES               =
+        {
+            ['Marid'] =
+            {
+                16990289,
```

Comment:
Same as before, luckily there are no chigoes before this group of IDs:

```

-- ------------------------------------------------------------
-- Bhaflau Thickets (Zone 52)
-- ------------------------------------------------------------
...

INSERT INTO `mob_spawn_points` VALUES (16990289,'Chigoe','Chigoe',18,141.403,-18.403,299.224,69);
INSERT INTO `mob_spawn_points` VALUES (16990290,'Chigoe','Chigoe',18,140.637,-18.339,299.261,220);
INSERT INTO `mob_spawn_points` VALUES (16990291,'Chigoe','Chigoe',18,141.781,-18.189,298.570,78);
INSERT INTO `mob_spawn_points` VALUES (16990292,'Chigoe','Chigoe',18,159.025,-16.229,333.930,177);
INSERT INTO `mob_spawn_points` VALUES (16990293,'Chigoe','Chigoe',18,156.374,-15.863,345.795,118);
```

```
['Marid'] = GetTableOfIDs("Chigoe", 5),
['Grand_Marid'] = GetTableOfIDs("Chigoe", 5)
```

Reply:
updated all of these references to use GetTableOfID's as requested.